client/js/components/statement/assessmentTable/AssessmentTableFilter.vue

```diff
@@ -60,8 +60,7 @@
       class="c-at__controls o-sticky o-sticky--border space-stack-xs u-pt-0_5 u-pb-0_25"
       ref="header">
       <search-and-sorting
-        :search-term="searchTerm"
-        :has-changed-statements="hasChangedStatements" />
```

Comment:
why is this obsolet?

Reply:
SearchAndSorting.vue doesn't have a `hasChangedStatements` prop, so it was added as an attribute. I could not
find any usage of it, so I believe that it was added here accidentally and can be removed